packages/react-sdk/src/useDisplayName.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from 'react'
+import { useSyncAgent } from './useSyncAgent'
+import { useObservable } from './useObservable'
+import { useAction } from './internals/useAction'
+
+export const useDisplayName = (spaceId: string, userId: string) => {
+    const sync = useSyncAgent()
+    const member = useMemo(
+        () => sync.spaces.getSpace(spaceId).members.getMember(userId),
+        [sync, spaceId, userId],
+    )
+    const { data, ...rest } = useObservable(member?.observables.displayName)
+    return {
+        ...data,
+        ...rest,
```

Comment:
Here I'm spreading the observable return so we can get a good API like 
```ts
const { username } = useUsername(spaceId, userId)
```

If I don't spread, I would need to do 
```ts
const { data: { username } } = useUsername(spaceId, userId)
```
which doesn’t look good and formatters can format this into two lines, making it even worse